code/src/App.js

```diff
@@ -1,12 +1,34 @@
 import React from 'react';
 import data from './data.json';
+import Header from './components/Header'
+import Cover from './components/Cover'
+import Albumname from './components/Albumname';
+import Artistname from './components/Artistname';
+import "./index.css";
 
-console.log(data);
 
 export const App = () => {
   return (
-    <div>
-      Find me in src/app.js!
+    <div className='all-container'>
+      <div className="Header"> 
+       <Header /> 
+      </div>
+      <div className='music-container'>
+      {data.albums.items.map (item => {
+       return <>
+        <div className='theAlbums'>
+          <Cover key={item.id} data={item} img={item.images}/> 
+          <Albumname Albumname={item.name} urlAlbumname={item.name}/>
```

Comment:
When we are clicking the album title the link is broken :( It seems like the urlAlbumname variable gets the wrong data. Try this instead:
`item.external_urls.spotify`